.depend

```diff
@@ -1,7 +1,7 @@
-_build/examples/cstubs_structs/bindings.cmo : _build/src/ctypes/ctypes.cmi _build/src/cstubs/cstubs_structs.cmi
-_build/examples/cstubs_structs/bindings.cmx : _build/src/ctypes/ctypes.cmx _build/src/cstubs/cstubs_structs.cmx
 _build/examples/cstubs_structs/bindings_c_gen.cmo : _build/src/cstubs/cstubs_structs.cmi
 _build/examples/cstubs_structs/bindings_c_gen.cmx : _build/src/cstubs/cstubs_structs.cmx
+_build/examples/cstubs_structs/bindings.cmo : _build/src/ctypes/ctypes.cmi _build/src/cstubs/cstubs_structs.cmi
```

Comment:
It appears that this change to `.depend` is unnecessary changed.  If that's the case, could you please remove the change from the patch?